### PR TITLE
Revise `SELECT` w/ missing `FROM` and fix a `TIME` test

### PR DESCRIPTION
**Pull request by am357**
Issue #, if available: #22 

Description of changes:
- Fixes a typo in `TIME no space between keyword and quote` test
- Revised `SELECT` with missing `FROM`.

More details:

Although `SELECT` with missing `FROM` is not support by
SQL-92 but it's still supported by major SQL DBMS like
PostgresSQL and MySQL. Considering this, moving these
tests as under PartiQL namespace as success tests.
We need to decide further on how we need to proceed forward
with this in our Kotlin reference implementation since it's
currently disallowed. See [#210](https://github.com/partiql/partiql-lang-kotlin/issues/210)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-tests-data/success/syntax/query/select/select.ion [hunk 1]
```diff
@@ -121,3 +121,149 @@
...
+
+{
+    name: "nested SELECT",
+    statement: "SELECT * FROM (SELECT * FROM x WHERE b).a",
+    assert: {
+        result: SyntaxSuccess
+    }
+}
+
+partiql::[
```
Comment (alancai98) [inline partiql-tests-data/success/syntax/query/select/select.ion:249]: I think the above tests added in this file are repeated?

File: partiql-tests-data/fail/syntax/query/select/select.ion [hunk 2]
```diff
@@ -61,11 +61,3 @@
         result: SyntaxFail
     },
 }
-
-{
```
Comment (alancai98) [inline partiql-tests-data/fail/syntax/query/select/select.ion:65]: Btw for the point about optional `FROM` in `partiql-lang-kotlin`, there was an issue to track making it optional: https://github.com/partiql/partiql-lang-kotlin/issues/210. Agree with moving this test under the `partiql` namespace and making it a success test.

**Review by alancai98 — COMMENTED** [review]
Approve of the changes other than some tests being repeated.

File: partiql-tests-data/success/syntax/query/select/select.ion (hunk 1 above)
Comment (am357) [reply partiql-tests-data/success/syntax/query/select/select.ion:249]: Nice catch, not sure how they've added—removed.

File: partiql-tests-data/fail/syntax/query/select/select.ion (hunk 2 above)
Comment (alancai98) [reply partiql-tests-data/fail/syntax/query/select/select.ion:65]: Might also be worth porting some of the tests mentioned in that lang-kotlin issue:

> 2. SELECT event.id would return << {id: 'foo'} >>
3. SELECT VALUE event.id would return << 'foo' >>
4. SELECT event.id WHERE event.type = 'SAVE' would return << {id: 'foo'} >>
5. SELECT event.id WHERE event.type != 'SAVE' would return << >>

**Review by alancai98 — APPROVED** [review]

File: partiql-tests-data/fail/syntax/query/select/select.ion (hunk 2 above)
Comment (am357) [reply partiql-tests-data/fail/syntax/query/select/select.ion:65]: +1, added them in the new commit.

Comment (alancai98) [reply partiql-tests-data/fail/syntax/query/select/select.ion:65]: Cool, thanks for including them

**Review by alancai98 — APPROVED** [review]

**Pull request merged**